scripts/generate-inline-script-hash.js

```diff
@@ -24,14 +28,15 @@ fs.readFile("./www/index.html", "utf8", (error, data) => {
         validResults.push(sha256Script);
       }
     }
-    fs.readFile("./manifest.json", "utf8", (error, data) => {
+
+    fs.readFile("./src/webplugin/" + process.argv[2] + "/manifest.json", "utf8", (error, data) => {
       let json = JSON.parse(data);
       json.content_security_policy = "object-src 'self'; script-src 'self' 'unsafe-eval' blob:";
       for (let i = 0; i < validResults.length; i++) {
         json.content_security_policy += " '" + validResults[i] + "'";
         console.log("Added sha256 script hash: " + validResults[i]);
       }
-      fs.writeFileSync("./manifest.json", JSON.stringify(json, null, 4));
+      fs.writeFileSync("./src/webplugin/" + process.argv[2] + "/manifest.json", JSON.stringify(json, null, 4));
```

Comment:
I'm just noticing this now but the manifest.json is still in the source map yet could be changed each build. Shouldn't it therefore also not be a part of the build output? E.g. remove from source move to build?